src/stravalib/protocol.py

```diff
@@ -8,11 +8,14 @@
 import abc
 import functools
 import logging
+import os
+import time
 from collections.abc import Callable
 from typing import TYPE_CHECKING, Any, Literal, TypedDict
 from urllib.parse import urlencode, urljoin, urlunsplit
 
 import requests
+from dotenv import load_dotenv
```

Comment:
in case the user wants to use a dot file. this would be a new dependency however . we could make it optional? or we could just stick to os.get_environ